trillian/ctfe/configpb/config.proto

```diff
@@ -101,6 +101,15 @@ message LogConfig {
   // log's key and can't sign STHs. Consequently, the log operator must ensure
   // to channel source log's STHs into CTFE.
   bool is_mirror = 12;
+
+  // The Maximum Merge Delay (MMD) of this log in seconds. See RFC6962 section 3
+  // for definition of MMD. If zero, the log does not provide an MMD guarantee.
+  int32 max_merge_delay_sec = 14;
+  // The merge delay that the underlying log implementation is able/targeting to
+  // provide. This option is exposed in CTFE metrics, and can be particularly
+  // useful to catch when the log is behind but still has not violated the
+  // strict MMD limit.
+  int32 expected_merge_delay_sec = 15;
```

Comment:
Could you populate these fields in `trillian/integration/{demo,ct_integration_test,ct_lifecycle}.cfg` so any examples that users might copy from have sensible values?

Reply:
Done.